### PR TITLE
Update UserGuide

### PR DESCRIPTION
**Pull request by amalinasani**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -1,33 +1,94 @@
...
+Example of usage: 
+* `edit -t 3 revise for CS2113`
+
+#### Editing a module: `edit -m <module_code> <new_module_code>`
+Edit a module code in the scheduler.  
+Format: `edit -m <module_code> <new_module_code>`
 
-`todo n/Write the rest of the User Guide d/next week`
+Example of usage: 
+* `edit -m “CS2113” “CS2113T”`
```
Comment (SimJunYou) [inline docs/UserGuide.md]: Do we actually need quotes around the old and new module codes?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -1,33 +1,94 @@
...
-Adds a new item to the list of todo items.
+### Adding an item: `add <opt><args>`
+#### Adding a task: `add -t [-by]`
+Add a task to the scheduler.  
+Format: `add -t <task_name> [-by <deadline>]`
 
-Format: `todo n/TODO_NAME d/DEADLINE`
+Example of usage: 
+* `add -t read chapter 1`
+* `add -t read chapter 1 -by 12 Sep`
```
Comment (SimJunYou) [inline docs/UserGuide.md]: Can we actually use this format for date?

File: docs/UserGuide.md [hunk 3]
```diff
@@ -1,33 +1,94 @@
...
 
 ## Features 
 
 {Give detailed description of each feature}
+### Viewing help: `help`
+Shows a message with the list of available commands and functions.
 
-### Adding a todo: `todo`
-Adds a new item to the list of todo items.
+### Adding an item: `add <opt><args>`
```
Comment (SimJunYou) [inline docs/UserGuide.md]: Missing a space between opt and args

File: docs/UserGuide.md [hunk 4]
```diff
@@ -1,33 +1,94 @@
...
-{Give a product intro}
+Repository Assistant with a Versatile Interface (ra.VI) is a desktop app for managing tasks, deadlines, and notes, optimized for use via a Command Line Interface (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, ra.VI can manage your tasks faster than traditional GUI apps.
 
 ## Quick Start
 
 {Give steps to get started quickly}
 
 1. Ensure that you have Java 11 or above installed.
-1. Down the latest version of `Duke` from [here](http://link.to/duke).
+2. Down the latest version of `ra.VI` from [here](http://link.to/duke).
```
Comment (SimJunYou) [inline docs/UserGuide.md]: Download instead of down?

File: docs/UserGuide.md [hunk 5]
```diff
@@ -1,33 +1,94 @@
...
+Mark a task in the scheduler as done.  
+Format: `done <task_index>`
+
+Example of usage: 
+* `done 1`
 
-`todo n/Refactor the User Guide to remove passive voice d/13/04/2020`
+### Exiting the program: `bye`
+Exits ra.VI.  
+Format:`exit`
```
Comment (SimJunYou) [inline docs/UserGuide.md]: Space after colon. Also, the command is `bye` instead of `exit` no?

File: docs/UserGuide.md [hunk 6]
```diff
@@ -1,33 +1,94 @@
...
+Edit a module code in the scheduler.  
+Format: `edit -m <module_code> <new_module_code>`
 
-`todo n/Write the rest of the User Guide d/next week`
+Example of usage: 
+* `edit -m “CS2113” “CS2113T”`
+
+### Deleting an item: `del <opt><args>`
+#### Deleting a task: `del -t <task_index>`
+Remove a task from the scheduler.
```
Comment (SimJunYou) [inline docs/UserGuide.md:60]: Should add an explanation here that the `task_index` can be found from `list -t`

**Review by SimJunYou — CHANGES_REQUESTED** [review]
Some changes required

File: docs/UserGuide.md (hunk 2 above)
Comment (Aseanseen) [reply docs/UserGuide.md]: Format for the deadline is d-M-yyyy HHmm\
e.g. add -t read chapter 1 -by 12-9-2020 1400

File: docs/UserGuide.md (hunk 6 above)
Comment (Aseanseen) [reply docs/UserGuide.md:60]: Agreed, because the task index can only be found through list

**Comment by SimJunYou** [comment]
Seems like UG updates are included in the other PR? Closing this one.

**Pull request merged**